app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file = command.split()[1]
```

Comment:
Generally, it's not a great idea to call a function twice where you can call it once. And here you can:
```python
command, file, new_file = command.split()

Reply:
brilliant suggestion, thx
Fixed :)